server/api/playbook_runs.go

```diff
@@ -801,23 +802,54 @@ func (h *PlaybookRunHandler) reminderButtonUpdate(w http.ResponseWriter, r *http
 	ReturnJSON(w, nil, http.StatusOK)
 }
 
+// reminderButtonDismiss handles the DELETE /runs/{id}/reminder endpoint, called when a
+// user clicks on the reminder custom_update_status dismiss button
+func (h *PlaybookRunHandler) reminderDelete(w http.ResponseWriter, r *http.Request) {
+	playbookRunID := mux.Vars(r)["id"]
+	userID := r.Header.Get("Mattermost-User-ID")
+	var payload struct {
+		ChannelID string `json:"channel_id"`
+	}
+	if err := json.NewDecoder(r.Body).Decode(&payload); err != nil {
+		h.HandleError(w, err)
+		return
+	}
+
+	h.removeReminderPost(w, userID, playbookRunID, payload.ChannelID)
+
+	w.WriteHeader(http.StatusNoContent)
+}
+
 // reminderButtonDismiss handles the POST /runs/{id}/reminder/button-dismiss endpoint, called when a
 // user clicks on the reminder interactive button
 func (h *PlaybookRunHandler) reminderButtonDismiss(w http.ResponseWriter, r *http.Request) {
+	playbookRunID := mux.Vars(r)["id"]
+	userID := r.Header.Get("Mattermost-User-ID")
 	requestData := model.PostActionIntegrationRequestFromJson(r.Body)
 	if requestData == nil {
 		h.HandleErrorWithCode(w, http.StatusBadRequest, "missing request data", nil)
 		return
 	}
 
-	playbookRunID, err := h.playbookRunService.GetPlaybookRunIDForChannel(requestData.ChannelId)
+	h.removeReminderPost(w, userID, playbookRunID, requestData.ChannelId)
+
+	ReturnJSON(w, nil, http.StatusOK)
+}
+
+func (h *PlaybookRunHandler) removeReminderPost(w http.ResponseWriter, userID, playbookID, channelID string) {
```

Comment:
I think this is a typo, right?
```suggestion
func (h *PlaybookRunHandler) removeReminderPost(w http.ResponseWriter, userID, playbookRunID, channelID string) {
```

Reply:
Ahhh, it strikes again! Thanks for catching that.